keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/admin/RealmsAdminResource.java

```diff
@@ -11,25 +14,68 @@
 import org.keycloak.services.resources.admin.permissions.AdminPermissionEvaluator;
 import org.keycloak.services.resources.admin.permissions.AdminPermissions;
 
+import javax.persistence.EntityManager;
+import javax.ws.rs.GET;
 import javax.ws.rs.NotFoundException;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
 import javax.ws.rs.core.Context;
 import javax.ws.rs.core.HttpHeaders;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
 
 public class RealmsAdminResource extends org.keycloak.services.resources.admin.RealmsAdminResource {
+    private ApiConfig apiConfig;
 
-    public RealmsAdminResource(AdminAuth auth, TokenManager tokenManager, KeycloakSession session) {
+    public RealmsAdminResource(AdminAuth auth, TokenManager tokenManager, KeycloakSession session, ApiConfig apiConfig) {
         super(auth, tokenManager);
         this.session = session;
         this.clientConnection = session.getContext().getConnection();
+        this.apiConfig = apiConfig;
+    }
+
+    @Path("terms-of-use-denied-users")
```

Comment:
it is not really denied-uers but not accepted because it may also be users who have not performed any connection.
It is a detail so not sure if it need to be renamed, I let you decide